parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -41,7 +41,7 @@ installDeps() {
     packages+=" cgroup-lite ceph-common glusterfs-client"
     if [[ $UBUNTU_RELEASE == "18.04" ]]; then
       disableTimeSyncd
-      packages+=" ntp ntpstat"
+      packages+=" ntp ntpstat chrony"
```

Comment:
we definitely wanna keep installing ntp and ntpstat when we're using chrony?

Reply:
Given that all previous images already came with ntp, ntpstat pre-installed, I think it is safer to keep them here?

Also, ntp and ntpstat may come handy in the future for debugging stuff in airgap cloud, but I will let you make a decision on whether to keep ntp ntpstat in golden image or not.

Reply:
Makes sense to me, let's start by continuing to include ntp (and ntpstat) and see if any gremlins show up when we enable chrony along with it